src/config.ts

```diff
@@ -10,33 +10,82 @@ import {duplicate, mergeDeep} from './util';
 import {VgRangeScheme} from './vega.schema';
 
 export interface CellConfig {
+  /**
+   * The default width of the single plot or each plot in a trellis plot when the visualization has a continuous (non-ordinal) x-scale or ordinal x-scale with `rangeStep` = `"fit"`.
```

Comment:
FYI, `rangeStep` = `"fit"` are now replaced with `rangeStep` = `null`.  (`"fit"` would no longer work.) Please help update all of them :)